test/fixtures/doc-yaml/user-guide/downward-api/dapi-pod.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
 spec:
   containers:
     - name: test-container
-      image: busybox
+      image: {{.BusyBoxImage}}
```

Comment:
I'm wary of turning what appear to be example files for docs into golang templates, do we know these files are used _only_ by e2e tests?

Reply:
Difference being those end in `.yaml.in` instead of `.yaml` so I don't expect them to be usable yaml out of the box.  Rename and we can find out if anything breaks.